filters/apiusagemonitoring/pathinfo.go

```diff
@@ -46,13 +59,19 @@ func (s pathInfoByRegExRev) Len() int           { return len(s) }
 func (s pathInfoByRegExRev) Less(i, j int) bool { return s[i].Matcher.String() > s[j].Matcher.String() }
 func (s pathInfoByRegExRev) Swap(i, j int)      { s[i], s[j] = s[j], s[i] }
 
-type metricNames struct {
-	GlobalPrefix            string
+type endpointMetricNames struct {
+	EndpointPrefix          string
 	CountAll                string
-	CountPerStatusCodeRange [5]string
+	CountPerStatusCodeRange [6]string
 	Latency                 string
 }
 
+type clientMetricNames struct {
+	CountAll                string
```

Comment:
are these objects marshalled? If not then we don't need to capitalize their names.

Reply:
i see this to be recurring case. We only need capitalized field names, when: 1. the enclusing structure is exported, or 2. we want to make the fields accessible for reflection e.g. json marshalling

Reply:
Absolutely right. Checking/fixing that right away.

Reply:
fixed in f1a2061a1c5fdc885eec9ef363ecfc7ad37dd243